### ISSUE TITLE
Tasks for week of 3/4 - 3/11

### ISSUE THREAD

**Issue by michaelchughes** [tufts-ai-robotics-group/polycraft-novelty-detection#1]
## Data loader

Here's the template code that we'll want to mimic for each dataset.

```
def create_data_generators_for_{DATASET_NAME}(
    	batch_size=1, shuffle=True,
    	classes_to_include=None):
    """ Create train/valid/test loaders for this dataset

    Args
    ----
    batch_size (int, optional)
    	batch_size for all data loaders. Defaults to 1.

    shuffle (bool, optional)
    	Whether we randomize the order of the datasets. Defaults to True.

    Returns
    -------
    train_loader : Pytorch DataLoader
		Contains batches of (N_CHANNELS, IM_SIZE, IM_SIZE) images for training.
		with pixel intensity values scaled to float between 0.0 and 1.0.
		Only images of the requested class labels will be present.
    valid_loader : Pytorch DataLoader
    	As above, for validation set
    test_loader  : Pytorch DataLoader
    	As above, for test set
“””
```

## TODOs for @PatrickFeeney 

* [ ] add classes_to_include functionality for MNIST
* [ ] do training on MNIST with some subset of classes (e.g. consider images of type "0" and "1" as "standard", leave others as novel)

## TODOs for Sarah

* [ ] Figure out way to parse Polycraft 256x256 images as 32x32 patches. Pytorch likely has code to do this already
* [ ] make data loaders for a big version of polycraft data, with config option to point at local location of the box data
* [ ] add structured data versions of the json labels for each image
